sdk/storage/azure-storage-blob/azure/storage/blob/aio/_upload_helpers.py

```diff
@@ -31,7 +31,7 @@
 
 if TYPE_CHECKING:
     from datetime import datetime # pylint: disable=unused-import
-    LeaseClient = TypeVar("LeaseClient")
+    BlobLeaseClient = TypeVar("BlobLeaseClient")
```

Comment:
Was this var unused in the rest of the file?
If it is, don't worry about it, probably just indicates we need a more thorough mypy pass - but just checking that no other instances needed renaming.

Reply:
Looks like it is unused. I can remove it when I make my docstring PR